### PR TITLE
OSDOCS-4383: BYOKMS for ROSA Clusters

### PR DESCRIPTION
**Pull request by bmcelvee**
Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/OSDOCS-4383

Link to docs preview:
OpenShift Cluster Manager: https://53655--docspreview.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.html#rosa-sts-creating-cluster-customizations-ocm_rosa-sts-creating-a-cluster-with-customizations

CLI: https://53655--docspreview.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.html#rosa-sts-creating-cluster-customizations-cli_rosa-sts-creating-a-cluster-with-customizations

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc [hunk 1]
```diff
@@ -33,7 +33,17 @@ link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[AWS Share
 ====
 To successfully install ROSA clusters, use latest version of the ROSA CLI.
 ====
-* If you are using a customer-managed AWS Key Management Service (KMS) key for encryption, you have created a symmetric KMS key and you have the key ID and Amazon Resource Name (ARN). For more information about creating AWS KMS keys, see link:https://docs.aws.amazon.com/kms/latest/developerguide/create-keys.html[the AWS documentation].
+* If you want to use a customer managed AWS Key Management Service (KMS) key for encryption, you must create a symmetric KMS key. You must provide the key ID and Amazon Resource Name (ARN) when creating your cluster. To create a customer managed KMS key, follow the procedure for link:https://docs.aws.amazon.com/kms/latest/developerguide/create-keys.html#create-symmetric-cmk[Creating symmetric encryption KMS keys].
```
Comment (mjlshen) [inline modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc:36]: My understanding is that we only ask for the ARN
```suggestion
* If you want to use a customer managed AWS Key Management Service (KMS) key for encryption, you must create a symmetric KMS key. You must provide the Amazon Resource Name (ARN) when creating your cluster. To create a customer managed KMS key, follow the procedure for link:https://docs.aws.amazon.com/kms/latest/developerguide/create-keys.html#create-symmetric-cmk[Creating symmetric encryption KMS keys].
```

File: modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc [hunk 2]
```diff
@@ -139,6 +149,7 @@ $ aws kms get-key-policy --key-id <key_id_or_arn> --policy-name default --output
                     "arn:aws:iam::<aws-account-id>:role/ManagedOpenShift-Installer-Role",
                     "arn:aws:iam::<aws-account-id>:role/ManagedOpenShift-Worker-Role",
                     "arn:aws:iam::<aws-account-id>:role/ManagedOpenShift-ControlPlane-Role"
```
Comment (mjlshen) [inline modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc:151]:
```suggestion
                    "arn:aws:iam::<aws-account-id>:role/ManagedOpenShift-ControlPlane-Role",
```

File: modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc [hunk 3]
```diff
@@ -159,6 +170,7 @@ $ aws kms get-key-policy --key-id <key_id_or_arn> --policy-name default --output
...
```
Comment (mjlshen) [inline modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc:172]:
```suggestion
                    "arn:aws:iam::<aws-account-id>:role/ManagedOpenShift-ControlPlane-Role",
```

**Review by mjlshen — COMMENTED** [review]
Looks good overall! Just a couple small things I noticed

File: modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc (hunk 1 above)
Comment (bmcelvee) [reply modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc]: Updated!

File: modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc (hunk 3 above)
Comment (bmcelvee) [reply modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc]: Updated!

File: modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc (hunk 2 above)
Comment (bmcelvee) [reply modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc]: Updated!

**Review by mjlshen — APPROVED** [review]
/lgtm

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Pull request merged**